### PR TITLE
Chart testing using `chart-testing-action`

### PR DESCRIPTION
**Pull request by Gaardsholt**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
No testing or linting of the helm chart is done at the moment, which might lead to releasing a non-functional helm chart.

Issue Number: N/A

## What is the new behavior?
Testing and linting is being done, to ensure that the helm chart is actually workable.

## Other information
I had to change the folder structure in order to get it working, no other changes has been made.

### CODE REVIEW

File: charts/kcale/Chart.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: kscale
 description: A Helm chart for kscale deployment
 type: application
-version: 0.0.1
+version: 0.0.2
```
Comment (OrKarstoft) [inline charts/kcale/Chart.yaml:5]: This isn't necessary because the action will automatically replace this with the correct value.

Comment (Gaardsholt) [reply charts/kcale/Chart.yaml]: it was the linting that wanted me to do it 😮 
I have reverted this and changed the linting.

**Comment by Gaardsholt** [comment]
> Why is it necessary to move the chart into an additional folder? `charts/kscale` instead of just charts?

Because I couldn't get it working any other way

**Review by OrKarstoft — APPROVED** [review]
I think we should try it. What's the worst that could happen?

**Pull request merged**